README.md

```diff
@@ -123,7 +123,9 @@ Currently documentation is available for the following systems:
 - [HEBBE](docs/hebbe.md)
 - [HKI](docs/hki.md)
 - [ICR_DAVROS](docs/icr_davros.md)
+- [IFB](docs/ifb-core.md)
```

Comment:
Thanks for adding IFB

Reply:
good spot on the typo

Reply:
But I made a mistake , it should be ifb_core.md... I corrected.

Reply:
That's what I meant, I saw your commit `correct file name IFB`